CONTRIBUTING.md

```diff
@@ -31,17 +31,17 @@ and detailed in your report, please ensure to specify at least the following:
     * A pull request with failing JUnit test case is most preferred, although it's OK to paste
       the test case into the issue description.
 
-You can consider a bug: some behavior that worked before and now it does not; a violation of the
+You can consider a bug: some behaviour that worked before and now it does not; a violation of the
 JDBC spec in any form, unless it's stated otherwise as an extension.
 
-It's hard, but possible, to introduce some breaking changes to the driver every major version update,
-so, please read carefully the changelog and test thoroughly for any potential problem with your app.
+In the unlikely event that a breaking change is introduced in the driver we will update the major version.
+We will document this change but please read carefully the changelog and test thoroughly for any potential problems with your app.
 What is not acceptable is to introduce breaking changes in the minor or patch update of the driver,
-if you found a regression in a minor o patch update, please fill an issue.
+If you find a regression in a minor patch update, please report an issue.
 
-Not every bug report needs to be of code, some can be documentation erratum that can be improved as
-well, the website source code along with the documentation is under **docs**, your are welcome to
-report issues and send pull request too.
+Bug reports are not isolated only to code, errors in documentation as well as the website source 
+code located in the **docs** directory also qualify. You are welcome to report issues and send a 
+pull request on these as well. [skip ci] can be added to the commit message to prevent Travis-CI from building for docs only changes
```

Comment:
>  [skip ci] can be added to the commit message to prevent Travis-CI from building for docs only changes

sounds weird to me

Maybe:
> For changes that only touch documentation, **[skip ci]** can be added to the commit message or pull request to prevent Travis-CI from building jobs.